Source/WTF/wtf/Vector.h

```diff
@@ -745,8 +745,9 @@ class Vector : private VectorBuffer<T, inlineCapacity, Malloc> {
                 if (auto item = valueGenerator(i))
                     unsafeAppendWithoutCapacityCheck(WTFMove(*item));
                 else
-                    return;
+                    continue;
```

Comment:
Are you sure no one is accidentally depending on "first `nullopt` stops the iteration" behavior?

Reply:
I will try and find out. It would result in correct behavior but I guess your point is that it may be slower.

Reply:
Oh actually, this can lead to trouble if the lambda doesn't expect to get called with future indexes. I will rework the PR.